concrete/src/Csv/Export/PageActivityExporter.php

```diff
@@ -0,0 +1,90 @@
+<?php
+namespace Concrete\Core\Csv\Export;
+
+use Concrete\Core\Attribute\Category\NullCategory;
+use Concrete\Core\Page\Page;
+use Concrete\Core\Attribute\ObjectInterface;
+use Concrete\Core\Localization\Service\Date;
+use League\Csv\Writer;
+
+class PageActivityExporter extends AbstractExporter
+{
+    /**
+     * @var \Concrete\Core\Localization\Service\Date
+     */
+    protected $dateService;
+
+    /**
+     * Initialize the instance.
+     *
+     * A dummy category is used to prevent that all kind of page
+     * attributes values are added to the CSV as well.
+     *
+     * @param \League\Csv\Writer $writer
+     * @param \Concrete\Core\Attribute\Category\NullCategory $nullCategory
+     * @param \Concrete\Core\Localization\Service\Date $dateService
+     */
+    public function __construct(Writer $writer, NullCategory $nullCategory, Date $dateService)
```

Comment:
Instead of adding `NullCategory`, what about overriding the `getAttributeKeysAndControllers` method (returning an empty array)?

Reply:
@mlocati I was in doubt of that. The AbstractExporter needs an implementation of `CategoryInterface`. I'd give a PageCategory implementation, but that felt wrong to me.

Reply:
> I'd give a PageCategory implementation, but that felt wrong to me.
Why? You are exporting pages after all...

Reply:
Well... it's more like a list of versions. I'll wait for @aembler's comments. I'm happy with both.

Reply:
Yeah, this is interesting – my apologies, l I didn't really look closely enough at AbstractExporter. I thought it was much more lightweight. This AbstractExporter class looks pretty opinionated. Let's not extend it after all. You're right – you're not actually doing anything with attributes in this export, so I'm not sure this class is going to get you anything anyway.

Reply:
Well, another solution could be to create a new CSV exporter abstract class that export an object without attributes: see https://github.com/mlocati/concrete5/commit/d25d59d64257eb459362142b7fd0910f6c427ae0

Reply:
And an even better solution could be to simply let AbstractExporter accept `null` as the attribute category (see https://github.com/mlocati/concrete5/commit/ac1d4b1aa4a7d4c6b7cd2a2f2da54eedb5b2a0c3).
This is not a problem for people that are already using this class: if they use it, they already pass an attribute category to the class constructor, so there's no BC problem.

Reply:
@mlocati Thanks for your help on this. 👍 Can you explain why you prefer the `null` solution more than the ObjectExporter solution?

Reply:
Because I'm thinking about refactoring the CSV exporter structure like this:
- `abstract class Exporter` which allows to export any kind of data
- `abstract class ObjectExporter extends Exporter` which allows to export attributed object, making the category optional (it will be the AbstractExporter of https://github.com/mlocati/concrete5/commit/ac1d4b1aa4a7d4c6b7cd2a2f2da54eedb5b2a0c3)
- `@deprecated abstract class AbstractExporter extends ObjectExporter ` which will be an empty class just for BC compatibility

My previous suggestion (https://github.com/mlocati/concrete5/commit/d25d59d64257eb459362142b7fd0910f6c427ae0) would introduce another class, making everything a bit more complicated without any reason.